core/include/thread.h

```diff
@@ -11,6 +11,100 @@
  * @ingroup     core
  * @brief       Support for multi-threading
  *
+ * Priorities
+ * ==========
+ *
+ * As RIOT is using a fixed priority @ref core_sched "scheduling algorithm",
+ * threads are scheduled based on their priority. The priority is fixed for
+ * every thread and specified during the threads creation by the `priority`
+ * parameter.
+ *
+ * The lower the priority number, the higher the priority of the thread,
+ * with 0 being the highest possible priority.
+ *
+ * The lowest possible priority is @ref THREAD_PRIORITY_IDLE - 1. The exact value
+ * depends on the platforms architecture, e.g. it would be 30 in 32-bit systems,
+ * and 14 in 16-bit systems.
+ *
+ * @note Assigning the same priority to two or more threads is usually not a
```

Comment:
How about...
```
A thread in RIOT may run until it yields (@ref
 *       thread_yield) or another thread with higher priority is runnable (@ref
 *       STATUS_ON_RUNQUEUE) again. Multiple threads with the same priority
 *       will therefore be scheduled cooperatively: when one of them is running,
 *       all others with the same priority depend on it to yield (or be interrupted
 *       by a thread with higher priority).
 *       This may make it difficult to determine when which of them gets
 *       scheduled and how much CPU time they will get.
```
?